operators/pkg/examagent/options.go

```diff
@@ -0,0 +1,104 @@
+// Copyright 2020-2022 Politecnico di Torino
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package examagent
+
+import (
+	"errors"
+	"flag"
+	"fmt"
+	"net"
+	"strings"
+
+	"k8s.io/klog/v2"
+
+	"github.com/netgroup-polito/CrownLabs/operators/pkg/utils/restcfg"
+)
+
+type options struct {
+	AllowedIPs       string
+	TemplatesNS      string
+	BasePath         string
+	ListenerAddr     string
+	PrintRequestBody bool
+	ipNets           []*net.IPNet
+}
+
+// Options object holds all the instanceset parameters.
+var Options options
+
+// Initialize flags and associate each parameter to the given options object.
+func (o *options) Init() {
+	flag.StringVar(&o.ListenerAddr, "address", ":8888", "[address]:port of the landing server")
+	flag.StringVar(&o.TemplatesNS, "templates-namespace", "", "Namespace of CrownLabs Templates that will be listed")
```

Comment:
This is probably just namespace, since everything is inside that (both instances and templates)